.github/codecov.yml

```diff
@@ -0,0 +1,8 @@
+fixes:
```

Comment:
oops, this shouldn't be here. Might need to `git fetch --all; git rebase --interactive origin/master; git push --force-with-lease` and ensure only the relevant commits are in here.

Reply:
Hi @MikeMcQuaid , I got a little confused about why there are so many commits appearing here as well. What I did was `git merge master` on my feature branch which I thought was a normal practice before we merge into master.

Reply:
@lionellloh Probably best to stick with `git merge origin/master` to ensure there's nothing extra on your `master`. In this case I'd `rebase`; it's a little more obvious locally what'll show up in the diff view.

Reply:
Ah ok, I think I understand what went wrong.